### DISCUSSION TITLE
✨Discord Rich Presence✨

### DISCUSSION THREAD

**Discussion by Insane-alastor1** [KRTirtho/spotube#2048] [Ideas]
![IMG_20241107_071640.jpg](https://github.com/user-attachments/assets/1deb294c-93af-41f2-b1ea-206bd823f0df)

Would you be able to create a [**Discord Custom Rich Presence**](https://discord.com/developers/docs/rich-presence/overview) when using the app and playing Music!? 🤗. It would be Great if You could.
 _Thank you❤️‍🩹 For making such a wonderful app..._

*Edit: Not available in Mobile Version

**Comment by Makifun** [comment]
This is already a feature...?

**Reply by Insane-alastor1** [reply]
@Makifun it's a feature in PC not in mobile🤗
